_posts/2023-09-30-Minutes-of_SCM-29.md

```diff
@@ -0,0 +1,77 @@
+- M ACTION: to send around both documents for OLS to the steering committee this week.
+- Secretary will submit invoices to get paid in advance for hours until the end of the year before the October shutdown. 
+
+
+## Debrief Unconference
+- Despite an agreement on the great success of the unconference, the steering committee concluded that the sustainable continuation of the unconference might incldue hiring a person dedicated to planning the next unconference.
+- A report will be written on the unconference that is shorter than last year.This year's unconference united 98 registrants compared to 141 from last year. 
+- Despite a slightly reduced number in participants compared to last year the level of conversation and discussion didn't seem to suffer.
+- The recordings of the unconference need to be moved to a YouTube location. Last year it was hosted by ARDC.
+- We do have over 540 attendees left on the Zoom Events - this could be used for another event if people were interested.
+- Licences might be offered to some international RSE group.
+
+## New setup for  unconference
+- Have one paid person to do the main organisation work who is included in a core group of organisators (maybe two chairs).
+- There is two years of documentation on how the conference should be run.
+- A potential new time for next unconference could be in April-May to use the Zoom licenses.
+
+## Debrief NZRSE conference
+
+## Secretary Position moving forward
+- Secretary going to be paid up until the end of the year (4 hours per month September to December total 16 hours dependent on funds). Hours might be used after the year if not used completely. Agreement from Rowland and Manodeep.
+- Secretary to create a handover document for new secretary. 
+
+
+## LinkedIn Organisation
+
+- RSE Australia does seem to have a LinkedIn Group, but do we have an organisation? Would this be beneficial?
+
+## Senior Advisory Board 
+
+- One of the panelists from the RSE conference was keen to be part of the senior advisory board.
+- This could be a starting point to reach out to people.
+- A short document might be created including expectations and tasks for Advisory Board members. This document might draw on what is already in the strategy document.
+
+## Face to Face meetings 
+
+- Would like to hand this over to JB.
+
+
+## Recruit new SC members
+
+- From the RSEAA meeting, could we identify and recruit some new SC members? 
+- All: Please add your recommendations, and let's identify who would be good to have onboard
+
+## New meeting time? 
+
+- Rescheduling of meeting time to ~2 pm Melbourne time to allow attendence for steering committee memmbers currently residing overseas.
+
+## RSE team leader workshop
+- Maybe give a workshop to leaders of the community (management for RSE's) (Rowland). This could also be a part of the unconference.
+- another potential option would be an RSE team leader AMA ask me anything.
```

Comment:
One of "Ask Me Anything" or "AMA" needs to be in "()"

Reply:
I pushed two commits to fix this.